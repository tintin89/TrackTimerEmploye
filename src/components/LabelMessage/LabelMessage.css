.labelMessage{
    background-color: salmon;
    color: red;
    font-family: 'Poppins', sans-serif;
    padding: 0.2rem;
    width: 90%;
    margin-top: 1rem;
    text-align: center;
}