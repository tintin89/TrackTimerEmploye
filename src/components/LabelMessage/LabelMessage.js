import React from 'react';
import './LabelMessage.css';

function LabelMessage(props) {
    return (
        <div className="labelMessage">
            {props.message}
        </div>
    )
}

export default LabelMessage
