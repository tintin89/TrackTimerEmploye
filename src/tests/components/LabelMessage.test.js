import { shallow } from 'enzyme';
import LabelMessage from '../../components/LabelMessage/LabelMessage';



describe('test of LabelMessage.js',()=>{
  const wrapper = shallow(<LabelMessage/>)

    test('Must show correctly',()=>{
     
        expect(wrapper).toMatchSnapshot();
    })
})