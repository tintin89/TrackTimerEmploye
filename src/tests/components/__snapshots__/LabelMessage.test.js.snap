// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test of LabelMessage.js Must show correctly 1`] = `
<div
  className="labelMessage"
/>
`;
